### ISSUE TITLE
Widget

### ISSUE THREAD

**Issue by VishnuSanal** [VishnuSanal/DialogMusicPlayer#33]
An app widget to control the currently playing music file (from: a request via e-mail!)

(an extension to #22)

**Comment by jmanan088** [comment]
@VishnuSanal Hey, i would like to try to implement this under hacktoberfest tag in java. Can you assign this me to?

**Comment by VishnuSanal** [comment]
@jmanan088 Thanks for showing interest, can you give me a summary of how you're planning to implement this? I feel like this is blocked by #22.

**Comment by jmanan088** [comment]
We can add the widget on home and lockscreen using by creating a new widget in project. Is that not possible? Sorry if I'm missing something, I'm new to this and just wanted to try it out.

**Comment by VishnuSanal** [comment]
@jmanan088 Yes, you can. But how about bringing in the functionality? i.e, handling play/pause & other controls, updation of the UI according to current progress etc.?

**Comment by jmanan088** [comment]
I'll have to search for that. Not sure.
